what,kind,bundleId,bundleVersion,description,time
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292168
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292169
error,log,,,"Unable to resolve module for:L/DatasetGenerator/src/version.py",1323851292170
activated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851292181
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292245
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292246
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292247
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292249
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292250
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292261
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292262
activated,view,org.python.pydev,2.0.0.2011040403,"org.python.pydev.navigator.view",1323851297718
executed,command,org.eclipse.ui,3.5.2.M20100120-0800,"org.eclipse.ui.file.refresh",1323851299340
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851299654
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851299657
deactivated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851301892
executed,command,org.eclipse.ui,3.5.2.M20100120-0800,"org.eclipse.ui.file.properties",1323851304835
activated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851304857
activated,editor,org.python.pydev,2.0.0.2011040403,"org.python.pydev.editor.PythonEditor",1323851309193
activated,view,org.python.pydev,2.0.0.2011040403,"org.python.pydev.navigator.view",1323851309218
deactivated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851310384
activated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851311898
deactivated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851336404
activated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851349454
deactivated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851353428
